cirq/google/xmon_device.py

```diff
@@ -195,6 +195,9 @@ def col(self, col: int) -> List[GridQubit]:
         """Returns the qubits in the given column, in ascending order."""
         return sorted(q for q in self.qubits if q.col == col)
 
+    def __repr__(self):
```

Comment:
This repr should be printing out a string that you could paste into ipython in order to get another instance of the current xmondevice. Not all xmon devices are foxtail, so this current repr is incorrect.

https://github.com/quantumlib/Cirq/pull/840 added repr's specific to foxtail and bristlecone, so it's no longer necessary to worry about that.

Example of a correct `__repr__` from `TrialResult`:

```python
    def __repr__(self):
        return ('TrialResult(params={!r}, '
                'repetitions={!r}, '
                'measurements={!r})').format(self.params,
                                             self.repetitions,
                                             self.measurements)
```

Reply:
I knew it should be like that, but all XmonDevice has is 
```
self._measurement_duration
self._exp_w_duration 
self._exp_z_duration 
self.qubits
```
Where do I get the device's name?

Reply:
The GridQubits generated by this class are not the same every time it's instantiated. How should I write a test for it?